dbms/src/Interpreters/Join.cpp

```diff
@@ -1997,6 +2082,13 @@ void Join::joinBlock(Block & block) const
 
         block.getByName(match_helper_name).column = ColumnNullable::create(std::move(col_non_matched), std::move(nullable_column->getNullMapColumnPtr()));
     }
+
+    if (isCrossJoin(kind))
```

Comment:
Add some cross join tests?

Reply:
> Add some cross join tests?

https://github.com/pingcap/tiflash/blob/6076a21ccded80a11a740c96329588ec27718372/dbms/src/Flash/tests/gtest_join_executor.cpp#L394
There are some cross join tests here. I add a test for cross join with null table.